samplers/jaegerremote/sampler_remote.go

```diff
@@ -70,7 +70,7 @@ type samplerUpdater interface {
 // delegates to it for sampling decisions.
 type Sampler struct {
 	// These fields must be first in the struct because `sync/atomic` expects 64-bit alignment.
-	// Cf. https://github.com/uber/jaeger-client-go/issues/155, https://goo.gl/zW7dgq
+	// Cf. https://github.com/jaegertracing/jaeger-client-go/issues/155, https://goo.gl/zW7dgq
```

Comment:
For another PR. `goo.gl` shortened links will soon not be available anymore.
See https://developers.googleblog.com/en/google-url-shortener-links-will-no-longer-be-available/

Reply:
Done, modified to the original link.